pkg/ovs/openflow/interfaces.go

```diff
@@ -69,6 +70,10 @@ const (
 	DeleteMessage
 )
 
+const (
+	PacketInForTraceFlow = uint8(1)
```

Comment:
is this used anywhere?

Reply:
Yes I think so. I recommend removing this constant from this PR, it can be part of the trace flow PR.

Reply:
Sure, I will remove it from this PR. @gran-vmv Please add the constant in the traceflow PR.